alchemiscale/storage/statestore.py

```diff
@@ -919,14 +1052,43 @@ def get_taskqueue_tasks(
         else:
             return [ScopedKey.from_str(t["_scoped_key"]) for t in tasks]
 
-    def claim_taskqueue_tasks(
-        self, taskqueue: ScopedKey, claimant: str, count: int = 1
+    def get_taskhub_unclaimed_tasks(
```

Comment:
Ah, we do use it in tests; this is fine.

Reply:
I was anticipating it might be useful for tests as you saw, but also possibly if one wanted to query what tasks are still waiting on the TaskHub eg for checks on the total volume of work etc. Can always YAGNI it away if not needed.